test-bom/build.gradle

```diff
@@ -3,15 +3,15 @@ plugins {
 }
 
 dependencies {
-    api platform("org.junit:junit-bom:$junitVersion")
-    api platform("org.spockframework:spock-bom:$spockVersion")
+    api(platform(libs.boms.junit))
+    api(platform(libs.boms.spock))
     constraints {
-        api "io.kotest:kotest-runner-junit5-jvm:$kotestVersion"
-        api "io.kotest:kotest-assertions-core-jvm:$kotestVersion"
-        api "io.mockk:mockk:$mockkVersion"
-        api "org.mockito:mockito-core:$mockitoVersion"
-        api "org.mockito:mockito-junit-jupiter:$mockitoVersion"
-        api "org.assertj:assertj-core:$assertJVersion"
-        api "org.hamcrest:hamcrest:$hamcrestVersion"
+        api(libs.kotest.runner.junit5.jvm)
```

Comment:
Instead of adding constraints on `api`, those should be managed dependencies in the catalog:

```
managed-kotest-runner-junit5-jvm = ...
```

Then the `bom` plugin will automatically take care of adding those to the BOM.

Reply:
Fixed.